src/actions/pause/fetchScopes.js

```diff
@@ -64,13 +64,13 @@ export function fetchScopes() {
       frame.generatedLocation.sourceId
     );
 
-    if (generatedSourceRecord.get("isWasm")) {
+    if (generatedSourceRecord && generatedSourceRecord.get("isWasm")) {
       return;
     }
 
     const sourceRecord = getSource(getState(), frame.location.sourceId);
 
-    if (sourceRecord.get("isPrettyPrinted")) {
+    if (sourceRecord && sourceRecord.get("isPrettyPrinted")) {
```

Comment:
does this need to be here? I think we should take this out because it's unrelated

Reply:
Yeah, I'll pull it out into a separate issue.